src/jdk.jdwp.agent/share/native/libdt_socket/socketTransport.c

```diff
@@ -224,7 +224,7 @@ handshake(int fd, jlong timeout) {
          * We should really use snprintf here but it's not available on Windows.
```

Comment:
This Windows comment should be removed. You can probably get rid of the jio_snprintf comment below as well since it adds little value, and doesn't make sense to just mention it here and not elsewhere.